RELEASE.txt

```diff
@@ -171,7 +171,7 @@ DOCUMENTATION
 DEVELOPMENT
 -----------
 
-- List visible changes in the way SCons is developed
```

Comment:
Minor nit - now proposing to leave the descriptive item in each section in place as guidance, to be removed upon release (not sure if @bdbaddog agrees, it's just what I've been doing)

Reply:
Can try for this next release and see how much of a hassle it is. Seems reasonable.